CHANGES.md

```diff
@@ -70,6 +70,11 @@ queueing logic centralized.
 how workflows are restarted
 ([#4040](https://github.com/cylc/cylc-flow/pull/4040)).
 
+"Implicit"/"naked" tasks (tasks that do not have an explicit definition in
+`flow.cylc[runtime]`) are now disallowed by default
+([#4109](https://github.com/cylc/cylc-flow/pull/4109)). You can allow them by
+setting `flow.cylc[scheduler]allow implicit tasks` to `True`.
+
```

Comment:
We'll need to tidy the change log a little before release - some entries (like this) added outside of the useful "enhancements" and "fixes" sections.

Reply:
I wasn't sure if it was a "notable" change for putting above the others or not

Reply:
OK, we haven't done that in the past on cylc-flow. Probably a good idea, we'll just need to add the appropriate "notable changes" heading.